Bump dependencies and better docs (#5)
willgm/web-crypto-storage: README.md

```ts
// Create a new instance of the crypto storage and give it a key!
// Be aware that normally you would never hard code the key like in this example,

> // take a look at the FAQ to more information on how to handle crypto keys.

Author: `for more information*`
Thanks!

`On how to handle crypto keys section? *`
The FAQ is all about it, so I think there is no necessity to talk about sections.

This project is heavily inspired on [Tim Taubert](https://twitter.com/ttaubert)'s talk at JSConf EU: [Keeping secrets with JavaScript](https://www.youtube.com/watch?v=yf4m9LdO1zI), if you want to better understand what happens under the hood, then I highly recommend for you to watch it out.

### Crypto Key FAQ

#### What should I use as the crypto key?

> There is no perfect answer for this question, it really depends on your objective using this library and the application itself. If you do want to have a strong security, and make as difficult as possible to someone not authorized to access the data, then the crypto key should be unique by user, something strong that only the he knows and it is never stored anywhere out of his mind, as the **user password**. But it is not the only possible strategy, you can do any sort of server integration to obtain a crypto key, by session or any other thing that makes sense to your product logic or application platform.

Reviewer: There is no perfect answer to this question, it depends on your objective using this library and the application itself. If you do want to have strong security and make as difficult as possible to someone not authorized to access the data, then the crypto key should be unique by a user, something strong that only the user knows and it is never stored anywhere out of his mind, as the user password. But it is not the only one possible strategy, you can create a server integration to obtain a crypto key, by session, or any other thing that makes sense to your product logic or application platform.

Author: I went all the way through your paragraph to find the changes and adopted most of it, but I still kept some as they are or changed in a different way. Thanks!

#### Where should I store the crypto key?

If you really want to have your data protected, then the crypto key should never be stored in the browser, ideally should be something that you never store anywhere and just your user knows, like his password. But for some apps it doesn't make sense to use the user password or other user input as crypto key, then would be better to use some server integration to store and obtain the key when needed.

#### What about hard coded key at the app bundle?

The only scenario that it can make sense is if you just don't want to make obvious what you are storing on the browser, but you are not really worried if an advanced user go all the way through to decrypt the data. Please consider a better strategy that makes sense to your application.

#### How does this library handles and stores the crypto key?

> This library does not stores the crypto key anywhere, not even in memory, 'cause the first thing that this library does when receives the given key is to derivate it to a new one, that will be really used to encrypt/decrypt and kept in memory for this session. The new derived crypto key cannot be reverted to the original value, and is only kept in memory, so the original key will be necessary again at the next session, so the stored encrypted data can be decrypted again.

Reviewer 2: ```suggestion
This library does not store the crypto key anywhere, not even in memory, 'cause the first thing  this library does when it receives the given key is to derivate it to a new one, which will be used to encrypt/decrypt and kept in memory for this session. The new derived crypto key cannot be reverted to the original value, and is only kept in memory, so the original key will be necessary again at the next session, this way the stored encrypted data can be decrypted again.
```

Author: Thanks! Gonna  add those typo fixes!
